flowzone.yml

```diff
@@ -2278,7 +2280,7 @@ jobs:
     name: Standardise repository settngs
     runs-on: ${{ fromJSON(inputs.runs_on) }}
     timeout-minutes: ${{ fromJSON(inputs.jobs_timeout_minutes) }}
-    needs: [event_types]
+    needs: [protect_branch]
```

Comment:
We can't change this to depend on `protect_branch` because protect branch only runs on unmerged PRs (so `do_final` would never be true)

I think we can leave this as-is?